src/components/App.js

```diff
@@ -1,11 +1,27 @@
 import React from 'react';
+import {NavBar} from './NavBar.js'
+import {Hero} from './Hero.js'
+import {HeroPicture} from './HeroPicture.js'
 import StatueListContainer from './StatueListContainer.js'
```

Comment:
You've created the index so you don't have to do this deep linking.

Reply:
Is this what it should be like?
`import { Hero, NavBar } from "./index.js";`

Reply:
Yes.